spec/requests/memory_spec.rb

```diff
@@ -21,3 +21,26 @@ def count_instances_of(klass)
   end
 
 end
+
+describe "Memory Leak", :type => :request, :if => RUBY_ENGINE == 'jruby' do
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.